subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -511,6 +512,22 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_pdu *rx)
 		goto ull_scan_aux_rx_flush;
 	}
 
+	/* Determine the window size */
+	if (aux_ptr->offs_units) {
+		window_size_us = OFFS_UNIT_300_US;
+	} else {
+		window_size_us = OFFS_UNIT_30_US;
+	}
+
+	/* Calculate received aux offset we need to have ULL schedule a reception */
+	aux_offset_us = (uint32_t)PDU_ADV_AUX_PTR_OFFSET_GET(aux_ptr) * window_size_us;
+
+	/* Skip reception if invalid aux offset */
+	pdu_us = PDU_AC_US(pdu->len, phy, ftr->phy_flags);
+	if ((aux_offset_us + window_size_us) < (pdu_us + EVENT_MAFS_US)) {
```

Comment:
The calculation, `(aux_offset_us + window_size_us) < (pdu_us + EVENT_MAFS_US)`, is added twice in this. Would it make sense to add a macro or function for it?